### ISSUE TITLE
Mob Asura Strike (Chen Bio4) and Ghostring Card effect

### ISSUE THREAD

**Issue by slyx88** [rathena/rathena#3521]
* **rAthena Hash**: 

* **Client Date**: 2017-06-14

* **Server Mode**: Renewal

* **Description of Issue**: Asura strike is not reduced by ghostring card
  * Result: Asura damage, in the range between 14k -19k in both conditons (with GR card and without GR card)
  * Expected Result:The damage from this skill should be reduced due to level 1 ghost elemental bonus from GR card 
  * How to Reproduce: Summon chen and compare the asura damage before and after wearing armor carded with GR card 
  * Official Information: A Nova RO player tested it in kRO sakray.
https://www.novaragnarok.com/forum/bug-tracker/fixed-bugs/asura-from-chen-x-r448/
 
  

* **Modifications that may affect results**:

**Comment by slyx88** [comment]
Thank you for your respond and solution @MathReaper .
But from the battle_attr_fix, wouldn't that make all other mob skills become forced neutral?
Correct me if im wrong.I am still in beginner level in programming =D

**Comment by MathReaper** [comment]
Yeah, probably, you can do alot of things to correct this, like, make a if-statement only for the skill that you want. But, currently, the rAthena is not making any usage of `battle_attr_fix` for monsters I guess, so maybe every skill is being neutral applied? I don't know.
